cmd/tempo-query/tempo/plugin.go

```diff
@@ -89,7 +89,7 @@ func (b *Backend) GetTrace(ctx context.Context, traceID jaeger.TraceID) (*jaeger
 		return nil, fmt.Errorf("%s", body)
 	}
 
-	otTrace, err := otlp.NewProtobufTracesUnmarshaler().UnmarshalTraces(body)
+	otTrace, err := ptrace.NewProtoUnmarshaler().UnmarshalTraces(body)
```

Comment:
i'm pretty sure i know what the answer is, but i have to ask: does this upgrade provide anyway to get around this nonsense?

Reply:
Right, I didn't find a way around it :(